boards/egis/egis_et171/egis_et171_defconfig

```diff
@@ -0,0 +1,34 @@
+CONFIG_XIP=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Serial driver options
+CONFIG_SERIAL=y
+
+# GPIO driver options
+CONFIG_GPIO=y
+
+# Syscon driver options
+CONFIG_SYSCON=y
+
+# hwinfo driver options
+CONFIG_HWINFO=y
+CONFIG_HWINFO_ANDES=y
+
+# FPU (single precision) options
+CONFIG_FPU=y
+#CONFIG_FPU_SHARING=y
+CONFIG_FLOAT_HARD=y
+CONFIG_SINGLE_PRECISION_FPU=y
```

Comment:
The FPU feature should be selected at the SoC level.

Reply:
Makes sense